core_data_modules/cleaners/codes/kenya_codes.py

```diff
@@ -342,298 +342,388 @@ class KenyaCodes(object):
     WUNDANYI = "wundanyi"
     YATTA = "yatta"
 
+    # Kenya Wards
+    # Machakos County Wards, src: https://kenyayote.com/iebc-machakos-county-registered-voters-constituency-wards/
+    ATHI_RIVER = "athi river"
+    EKALAKALA = "ekalakala"
+    IKOMBE = "ikombe"
+    KATANGI = "katangi"
+    KANGUNDO_NORTH = "kangundo north"
+    KANGUNDO_CENTRAL = "kangundo central"
+    KANGUNDO_EAST = "kangundo east"
+    KANGUNDO_WEST = "kangundo west"
+    KITHIMANI = "kithimani"
+    KATHIANI_CENTRAL = "kathiani central"
+    KYELENI = "kyeleni"
+    KOLA = "kola"
+    KIBAUNI = "kibauni"
+    KALAMA = "kalama"
+    KINANIE = "kinanie"
+    KIVAA = "kivaa"
+    LOWER_KAEWA_KAANI = "lower kaewa/kaani"
+    MATUU = "matuu"
+    MATUNGULU_NORTH = "matungulu north"
+    MATUNGULU_EAST = "matungulu east"
+    MATUNGULU_WEST = "matungulu west"
+    MITABONI = "mitaboni"
+    MUA = "mua"
+    MUTITUNI = "mutituni"
+    MACHAKOS_CENTRAL = "machakos central"
+    MUMBUNI_NORTH = "mumbuni north"
+    MUVUTINI_KIIMA_KIMWE = "muvutini/kiima-kimwe"
+    MBIUNI = "mbiuni"
+    MAKUTANO = "makutano"
+    MASII = "masii"
+    MUTHETHENI = "muthetheni"
+    MASINGA_CENTRAL = "masinga central"
+    MUTHESYA = "muthesya"
+    MUTHWANI = "muthwani"
+    NDITHINI = "ndithini"
+    NDALANI = "ndalani"
+    SYOKIMAU_MULOLONGO = "syokimau/mulolongo"
+    TALA = "tala"
+    UPPER_KAEWA_IVETI = "upper kaewa/iveti"
+    WAMUNYU = "wamuyu"
+
+    # Machakos wards -> consituency map
+    # Todo: update the rest of the wards
+    WARD_TO_CONSTITUENCY_MAP = {
+            KIVAA: MASINGA,
```

Comment:
Indentation, here and in the next dict
```suggestion
    WARD_TO_CONSTITUENCY_MAP = {
        KIVAA: MASINGA,
```